### DISCUSSION TITLE
loggingRef

### DISCUSSION THREAD

**Discussion by frit0-rb** [kube-logging/logging-operator#1501] [Q&A] [answered]
Hi guys,

I need to understand better loggingRef, I look is necessary with two or more loggings to create fluentd and fluentbit, but I dont understand as much I want.

Can you help me?

**Comment by pepov** [comment, marked as answer]
Hey! A logging ref is something that groups Logging, Flow, Output and FluentbitAgent resources together in a single Domain, so that it won't collide with other resources.

I've just written a summary on the possible multi-tenancy options and a new feature providing more isolation has arrived in the 4.4 release, please have a look and let me know if that helped: https://kube-logging.dev/docs/whats-new/#multitenancy-with-namespace-based-routing
